docs/src/architecture/domain-model.md

```diff
@@ -38,27 +47,80 @@ Usual **devices** are **PLCs** (**P**rogrammable **L**ogic **C**ontrollers), **I
     **Sensors** and **Actuators** to the **device** as inputs or outputs
   * also the **Domain Application** can expose data as input or output (e.g. own _signals_ or _states_)
 
+## IoT Cloud Domain
+
+**thin-edge** is designed to be cloud agnostic, i.E. it does not stick to some specific IoT cloud.
+Potentialy clouds are [Cumulocity](https://www.softwareag.cloud/site/product/cumulocity-iot.html), [Azure](https://azure.microsoft.com) or [AWS](https://aws.amazon.com). A flexible _Mapper Concept_ allows to adapt thin-edge to other clouds as well.
+
+TODO: Maybe avoid cloud-names above, but focus on _mapper-concept_, since Azure and AWS are far less supported.
+
+* Each **device** is represented in the IoT cloud by an individual **device twin**
+* All information / data or functionality the device provides to the cloud appears in context of it's **device twin**
+* The focus of **thin-edge** is on two IoT Cloud aspects:
+  - 1) **Telemetry Data Handling**
+  - 2) **Device Management**
+
+### Telemetry Data Handling
+
+**Telemetry Data Handling** provides to manage data coming from device's sensors, or process data or signals coming from device's domain application.
+Those information are categorized as below:
+
+  * **Measurement**, is a single value or set of values
+    * could be be a mix of values produced by one or more sensors and values calculated by the device's domain application
+    * values could be a mix of numbers, strings or booleans
+    * has one timestamp given by the producer, or implicitly set as the current time of receiving at **thin-edge**
+    * releates to one **Metric**
+  * **Metric**, is a time-series of measurements
+    * contains a _source device_
+    * contains a _type name_
+    * optionally contains _units_ for the **measurements**
```

Comment:
> The source it not necessarily a device - it can be a process.

For me each measurement for a metric comes from a device. If the process on a device that produces measurements for a metric shall appear as individual source, then that process should act as a child-device.

```suggestion
  * **Metric**, is a time-series of measurements
    * relates to a _source device_
    * has a _type name_
    * optionally holds _units_ for the **measurements**
```

What do you think about that?

Reply:
> For me each measurement for a metric comes from a device. If the process on a device that produces measurements for a metric shall appear as individual source, then that process should act as a child-device.

That is not the case. `collectd` is a source of measurement that is not represented as a child device.

Reply:
Applied in new PR, see https://github.com/thin-edge/thin-edge.io/pull/1554#issuecomment-1303825437